### ISSUE TITLE
Refactor inactive-blog-filter.js to use Redis vs. files

### ISSUE THREAD

**Issue by humphd** [Seneca-CDOT/telescope#405]
We have some logic that is using files to deal with state, and we need to move that into Redis, so it can be distributed across nodes in production.

The various files that are being used in `inactive-blog-filter.js` should be refactored to use Sets in Redis, which we should be able to use to do set operations (unions, diffs, intersections, etc), and thereby keep our list of feeds up to date.

See the code in `storage.js` for some ideas on how to do this.  We'll need to add more code there for this use case.

**Comment by Silvyre** [comment]
Please allow me!

**Comment by Silvyre** [comment]
Stepping aside from this until #406 is resolved.

**Comment by haichuan0424** [comment]
Karla and I are going to do this.

**Comment by haichuan0424** [comment]
Going to take #270 instead. Feel free to take this.

**Comment by c3ho** [comment]
If someone is not taking this, I think I can work on it too, since there is some dependency from 462 on this.

**Comment by c3ho** [comment]
This depends on changes from #462

**Comment by c3ho** [comment]
```
async function check(feedUrl, redlistPath = 'feeds-redlist.json') {
  // Read redlist file
  try {
    const redListRaw = await fsPromises.readFile(redlistPath, 'utf-8');
    const redList = [].concat(JSON.parse(redListRaw));
    return redList.some(redItem => isRedlisted(feedUrl, redItem));
  } catch (error) {
    logger.error({ error }, `failed to read ${redlistPath}`);
    throw error;
  }
}
```

We're using URL here did you want to change to use a ```feedId``` here instead?

**Comment by humphd** [comment]
I want to get rid of this code completely, and use the the invalid key concept with a feed id like I mentioned in the other comments I left.

**Comment by c3ho** [comment]
Closing this as it was addressed by #630
